[#3] Docs : README.md 디렉토리 구조 수정 + eslint설정 파일 명시

Seonwu-K opened the pull request.

Seonwu-K committed:
[#3] Docs : README.md 디렉토리 구조 수정 + eslint설정 파일 명시

Seonwu-K commented:
이제 DB연결 이전에 할 수 있는 express 폴더 구조 세팅이 모두 끝났습니다.  

현재 작업 시점에 작성할 수 있는 파일이 없어 비어있는 폴더의 경우 커밋기록이 따로 남지 않기 때문에 현재 최신상태 브랜치에 접근하더라도 폴더가 없을 겁니다.

readme.md의 directory structure에서 폴더명과 구조 참고하여 필요한 파일을 내부에 작성해주시면 될 것 같습니다.

bagger3025 committed:
[#3] Feat : Add prettier and .vscode folder

- prettier를 dev 모드로 설치
- prettierrc 파일 추가
- 자동 설정을 위한 .vscode 폴더의 settings.json 추가

bagger3025 commented:
prettier도 추가하였습니다. 자동설정을 위한 .vscode 폴더도 추가하였습니다.
prettier 설정이나 eslint 설정 중 원하는 것이 있으면 수정하면 좋을 것 같습니다.

bagger3025 committed:
[#3] Feat : Edit prettier settings and README.md

- README.md에 .prettierrc.json 추가
- setting.json과 .prettierrc.json이 tab을 기본으로 하고 그 크기는 4칸이 되도록 세팅,
  기본 세팅이 무엇이든 4칸 탭이 됨

bagger3025 commented:
prettier 세팅을 탭+4칸으로 하였고, vscode 상에서 항상 4칸 탭으로 보이도록 세팅했습니다.

Seonwu-K commented:
이제 기본 세팅은 거의 완성된 듯 하여 이슈를 닫고 브랜치를 지우도록 하겠습니다.
이제부터 추가로 필요한 미들웨어나 라이브러리는 개발 기능과 연관된 경우가 대부분일 것으로 예상되어
해당 이슈에서 다루면 될 것 같습니다
